pkg/controller/provider/web/ocp/namespace.go

```diff
@@ -112,14 +112,14 @@ func (h NamespaceHandler) Link(p *api.Provider, m *model.Namespace) string {
 // REST Resource.
 type Namespace struct {
 	Resource
-	Object interface{} `json:"object"`
+	Object core.Namespace `json:"object"`
```

Comment:
By doing this, the object unmarshalled by the REST client will be the actual k8s object.  Same effect for the json rendered by the REST API but much more useful when using the go client in the other controllers..